### PR TITLE
docs: fix, add version support text

### PR DESCRIPTION
**Pull request by kosekaku**
**Docs**

add a text description for the  dhis2 version support
From the docs, there is no description of dhis2 versions  with support for the personal access tokens

### CODE REVIEW

File: src/user/personal-access-tokens.md [hunk 1]
```diff
@@ -13,18 +13,21 @@ it sends your username and password in clear text. It may be deprecated in
 future DHIS2 versions or made opt-in, meaning that basic authentication would
 need to be explicitly enabled in the configuration.
 
+#### Dhis2 supported versions
+
+To utilize the benefits of PATs, you will need to be on dhis2 2.37.X or later version.
```
Comment (Philip-Larsen-Donnelly) [inline src/user/personal-access-tokens.md]:
```suggestion
To utilize the benefits of PATs, you will need to be on DHIS version 2.37 or above.
```

File: src/user/personal-access-tokens.md [hunk 2]
```diff
@@ -13,18 +13,21 @@ it sends your username and password in clear text. It may be deprecated in
 future DHIS2 versions or made opt-in, meaning that basic authentication would
 need to be explicitly enabled in the configuration.
 
+#### Dhis2 supported versions
```
Comment (Philip-Larsen-Donnelly) [inline src/user/personal-access-tokens.md]:
```suggestion
#### DHIS2 supported versions
```

**Comment by kosekaku** [comment]
@Philip-Larsen-Donnelly 
any other thing needed from my side? Do I need to send a commit again?

**Comment by Philip-Larsen-Donnelly** [comment]
Hi @kosekaku 
No need for any other action. Thanks for the PR. 
I wanted to consider this a little :)
We don't usually indicate the versions where we introduce features. As you can imagine, that could get quite messy across the docs. We usually just rely on the fact that docs for the old versions do not contain the feature.

However, in this case perhaps we can make an exception, as it is quite a key security feature.

**Comment by larshelge** [comment]
We have a convention not to include version in the docs and instead infer from the overall docs DHIS 2 version. Closing for now.

**Pull request closed without merging**